parser.py

```diff
@@ -134,10 +162,52 @@ def extract_username(account_js_filename):
     account = read_json_from_js_file(account_js_filename)
     return account[0]['account']['username']
 
+def collect_tweet_id(tweet):
+    if 'tweet' in tweet.keys():
+        tweet = tweet['tweet']
+    return tweet['id_str']
+
+
+def collect_tweet_references(tweet, known_tweet_ids, counts):
+    if 'tweet' in tweet.keys():
+        tweet = tweet['tweet']
+    tweet_ids = set()
+    # Collect quoted tweets
+    if 'entities' in tweet and 'urls' in tweet['entities']:
+        for url in tweet['entities']['urls']:
+            if 'url' in url and 'expanded_url' in url:
+                expanded_url = url['expanded_url']
+                matches = re.match(r'^https://twitter.com/([0-9A-Za-z_]*)/status/(\d+)$', expanded_url)
```

Comment:
`r'^https://twitter.com/(\w+)/status/(\d+)$'` would be slightly more concise

Reply:
Yeah, that's probably right. I copied that part of the Regex from `convert_tweet`, so the same simplification should be ok there, too.